tinker/templates/admin/redirects/clear-redirects.html

```diff
@@ -1,40 +1,21 @@
+<div>
 {% if changed|length == 0 and deleted|length == 0 %}
-    <p>There are no changes or deletions to worry about. Good work!</p>
+    There are no changes or deletions to worry about. Good work!<br>
```

Comment:
Nitpick: `<br/>`

Reply:
Yeah I removed those because they didn't work in the test 🤷‍♂️

Reply:
Yeah, but it should be `<br/>` not `<br>`. That's the nitpick

Reply:
Somehow going through: html template > email > FP case creation, <br/> didn't carry over, but <br> did.

Reply:
Rip